hack/push-chart.sh

```diff
@@ -35,10 +42,14 @@ readonly k8s_registry="registry.k8s.io/kueue"
 readonly semver_regex='^v([0-9]+)(\.[0-9]+){1,2}(-rc\.[0-9]+)?$'
 
 image_repository=${IMAGE_REPO}
+image_repo_kueueviz_backend=${IMAGE_REPO_KUEUEVIZ_BACKEND}
+image_repo_kueueviz_frontend=${IMAGE_REPO_KUEUEVIZ_FRONTEND}
 app_version=${GIT_TAG}
 if [[ ${EXTRA_TAG} =~ ${semver_regex} ]]
 then
-	image_repository=${k8s_registry}/kueue
+	image_repository=${k8s_registry}/${IMAGE_NAME}
+	image_repo_kueueviz_backend=${k8s_registry}/${IMAGE_NAME_KUEUEVIZ_BACKEND}
+	image_repo_kueueviz_frontend=${k8s_registry}/${IMAGE_NAME_KUEUEVIZ_FRONTEND}
```

Comment:
I don't think we need these two variables, and IMAGE_REPO_KUEUEVIZ_BACKEND and IMAGE_REPO_KUEUEVIZ_FRONTEND either.

Reply:
I think we just need to change STAGING_IMAGE_REGISTRY variable (`us-central1-docker.pkg.dev/k8s-staging-images` or `registry.k8s.io/kueue`). Everything else should be the same. But I would prefer to fix it in separate PR.

Reply:
Yes, we have too many variables which can never change. I would propose to remove IMAGE_REPO_KUEUEVIZ_FRONTEND in a follow up from the main Makefile. I don't see a need to propagate it in this PR, or use at all.

Reply:
> Yes, we have too many variables which can never change. I would propose to remove IMAGE_REPO_KUEUEVIZ_FRONTEND (and the backend one) in a follow up from the main Makefile. I don't see a need to propagate it in this PR, or use at all.

Done